pkg/config/config_test.go

```diff
@@ -409,6 +410,741 @@ func TestNewCreateChannelTxFailure(t *testing.T) {
 	}
 }
 
+func TestSystemChannelGenesisBlock(t *testing.T) {
```

Comment:
nit: `TestNewSystemChannelGenesisBlock`

Reply:
Sorry about for the following four nits. I have fixed them all.